src/test/java/PersonTest.java

```diff
@@ -0,0 +1,42 @@
+import org.example.Man;
+import org.example.Woman;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class PersonTest {
+
+    @Test
+    public void testRegisterPartnership() {
+        Man evgeniy = new Man("Evgeniy", "Krivorog", 32);
+        Woman natalia = new Woman("Natalia", "Krivorog", 32);
+
+        evgeniy.registerPartnership(natalia);
+        assertEquals(natalia, evgeniy.getPartner());
+        assertEquals("Krivorog", evgeniy.getLastName());
```

Comment:
Very small amount of asserts, how about firstName, age for both woman and man. And also you do not check partner for woman and last name